frontend/components/commons/sidebar/TaskSidebar.vue

```diff
@@ -52,6 +53,9 @@ export default {
     currentTask() {
       return this.dataset.task;
     },
+    currentTaskHelp() {
+      return this.currentTask + "ViewInfo";
```

Comment:
it's more clean to do concatenation with backtick instead of "+" => replace with `${this.currentTask}ViewInfo`